app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1113,8 +1113,12 @@ def _validate_languages_versions(self, data):
             if NimbusExperiment.Version.parse(
                 min_version
             ) < NimbusExperiment.Version.parse(min_supported_version):
+
                 raise serializers.ValidationError(
-                    {"languages": "Languages are not supported for this version."}
+                    {
+                        "languages": "Languages min version required:"
```

Comment:
Good idea but

1) String formatting is preferred over string concatenation 
2) Perhaps slightly more expressive: `Language targeting is not supported for this application below version {version}`?

Reply:
yeah great point, flake was not happy with the too longs character, I split that up now and error message is improved now 🎉 , thank you @jaredlockhart 🙏 
![image](https://user-images.githubusercontent.com/104033388/173902544-dd103c87-f41e-486d-a2be-304410890dce.png)